toggles/webapp/toggles.ts

```diff
@@ -69,6 +69,12 @@ const toggles = {
       description:
         'Adds tabbed navigation to the works page, for switching between work, item and related content',
     },
+    {
+      id: 'itemWorkLink',
+      title: 'Item page: Work page link',
+      initialValue: false,
+      description: 'Repositiions and changes the style of the work page link',
```

Comment:
although it's implied by the toggle name, might be good to be explicit about which work page link:

```suggestion
      description: 'Tries an alternative style for the link from the item page to the corresponding work',
```